builds/jazz-build-module/service-configuration-data-loader.groovy

```diff
@@ -23,15 +23,51 @@ echo "Service configuration module loaded successfully"
 /**
  * Initialize the module
  */
-def initialize(config, role_arn, region, role_id, jenkins_url, current_environment, service_name, utilModule) {
-  config_loader = config
-  setRoleARN(role_arn)
-  setRegion(region)
-  setRoleId(role_id)
-  setJenkinsUrl(jenkins_url)
-  setCurrentEnvironment(current_environment)
-  setServiceName(service_name)
-  setUtilModule(utilModule)
+def initialize(configLoader, role_arn, region, accountId, jenkins_url, current_environment, service_name, utilModule) {
+    config_loader = configLoader
+    setRoleARN(role_arn)
+    setRegion(region)
+    setAccountId(accountId)
+    setJenkinsUrl(jenkins_url)
+    setCurrentEnvironment(current_environment)
+    setServiceName(service_name)
+    setUtilModule(utilModule)
+}
+
+def getApigatewayInfoCore(domain, stage){
```

Comment:
Same method written in apigateway module. Please reuse it

Reply:
done